invenio_app_ils/cli.py

```diff
@@ -607,6 +613,130 @@ def persist(self):
         return recs
 
 
+class VendorGenerator(Generator):
```

Comment:
This `cli.py` is becoming massive too, we might need to break it in blocks eventually... :)

Reply:
Completely agree. `cli.py` should only have the commands imo and extract the functionality into api. Would be a good candidate for the refactor that was planned. :)